File: frontend/bookstore-app/src/types/cuota.ts
export interface CuotaListado {
  id: number;
  comprobanteId: number;
  numeroComprobante?: string;
  fechaComprobante?: string;
  clienteId: number;
  clienteNombre?: string;
  zonaId?: number;
  zonaNombre?: string;
  fechaCuota?: string;
  importe: number;
  importePagado: number;
  estado?: string;
  esCuotaCero: boolean;
}

export interface UpdateImportePagadoDto {
  importePagado: number;
  esCuotaCero: boolean;
  comprobanteId?: number;
}

File: backend/BookstoreAPI/Models/Cuota.cs
namespace BookstoreAPI.Models
{
    public class Cuota
    {
        public int Id { get; set; }
        public int? Comprobante_Id { get; set; }
        public DateTime? Fecha { get; set; }
        public decimal? Importe { get; set; }
        public decimal? ImportePagado { get; set; }
        public string? Estado { get; set; }
    }
}
